content/doc/developer/plugin-development/writing-an-scm-plugin.adoc

```diff
@@ -0,0 +1,29 @@
+---
+title: Writing an SCM Plugin
```

Comment:
https://github.com/jenkinsci/scm-api-plugin/blob/master/docs/consumer.adoc & https://github.com/jenkinsci/scm-api-plugin/blob/master/docs/implementation.adoc are much newer and more authoritative and also cover usage of the legacy core API.